### PR TITLE
Add github action to build/push dependency image.

### PR DESCRIPTION
**Pull request by nkraemer-sysdig**
Devops is not currently supporting secrets for github actions, and since this is a public repo (it was forked from a public repo) we cannot use the github action secrets. Switching to Jenkins instead: https://sysdig-jenkins.internal.sysdig.com/view/Secure/job/kube-bench/

Leaving the github action yaml file in case we want to switch in the future (The workflow is disabled)

### CODE REVIEW

**Review by yathi-sysdig — APPROVED** [review]

File: .github/workflows/build-and-push-dependency.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Build and push dependency
+on:
+  create:
+    tags: [ "*" ]
```
Comment (yathi-sysdig) [inline .github/workflows/build-and-push-dependency.yaml:4]: this would run no every change I think.. even if the change is not related to benchmarks

Comment (yathi-sysdig) [reply .github/workflows/build-and-push-dependency.yaml:4]: ah nevermind.. thught this was secure-backend :)

Comment (nkraemer-sysdig) [reply .github/workflows/build-and-push-dependency.yaml:4]: Yep, check the repo name =p

**Review by VadimZy — APPROVED** [review]
looks good

**Comment by nkraemer-sysdig** [comment]
Devops is not currently supporting secrets for github actions, and since this is a public repo (it was forked from a public repo) we cannot use the github action secrets. Switching to Jenkins instead: https://sysdig-jenkins.internal.sysdig.com/view/Secure/job/kube-bench/

**Review by yathi-sysdig — APPROVED** [review]
